policyengine_us/variables/gov/states/sc/tax/income/credits/college_tuition/sc_tuition_credit_eligible.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class sc_tuition_credit_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for the South Carolina Tuition Credit"
+    definition_period = YEAR
+    reference = (
+        "https://dor.sc.gov/forms-site/Forms/I319_2021.pdf#page=2",
+        "https://www.scstatehouse.gov/code/t12c006.php"
+        # South Carolina Legal Code | SECTION 12-6-3385 (B)(3)(b)
+    )
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.sc.tax.income.credits.college_tuition
+        total_college_hours = person("total_college_hours", period)
+        semesters_attended = person("college_semesters_attended", period)
+        required_hours = p.semester_hour_requirement * semesters_attended
+        meets_hours_requirement = total_college_hours >= required_hours
```

Comment:
Doesn't this imply `semesters_attended > 0`?
```suggestion
        return total_college_hours >= required_hours
```

Reply:
I need this to pass the test case where a student is ineligible for attending 0 semester and have 0 credit.

Reply:
How could they have `(total_college_hours > 0) & (college_semesters_attended == 0)`? Is it that they need to have completed a semester? If so let's rename the variable to `college_semesters_completed`.